blog/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.urls import path
+from .views import Index, PostDetailView, CommentCreateView
+
+urlpatterns = [
+    path("", Index.as_view(), name="index"),
+    path("posts/<int:pk>", PostDetailView.as_view(), name="post-detail"),
```

Comment:
it is generally considered good practice to include a trailing slash at the end of your URLs.